swagger_ui_bundle/__init__.py

```diff
@@ -12,13 +12,12 @@ def get_path(rel):
     )
 
 
-swagger_ui_2_2_10_path = get_path("vendor/swagger-ui-2.2.10")
-
 swagger_ui_3_52_0_path = get_path("vendor/swagger-ui-3.52.0")
+swagger_ui_4_4_0_path = get_path("vendor/swagger-ui-4.4.0")
 
 # latest major versions
-swagger_ui_2_path = swagger_ui_2_2_10_path
 swagger_ui_3_path = swagger_ui_3_52_0_path
+swagger_ui_4_path = swagger_ui_4_4_0_path
 
 # default to swagger 3
 swagger_ui_path = swagger_ui_3_path
```

Comment:
Kept v3 as default for now to limit impact of change.
But since a major version update is needed anyway, we might want to change it to v4 already.